hera_pspec/tests/test_uvpspec.py

```diff
@@ -467,20 +504,20 @@ def test_combine_uvpspec(self):
         # setup uvp build
```

Comment:
How are `r_params` dealt with by `combine_uvpspec`, e.g. when two UVPSpec objects are added together?

Reply:
(or `select` for that matter)

Reply:
combine_uvpspec now combines` r_params` but refuses to combine two uvpspec objects with a conflict in `r_params` (dictionaries are not equal for same baseline in two different uvpspec objects). 

`select` trims `r_params` to only contain entries that refer to baselines in the down selected boject.